services/frontend-v3/src/i18n/en_CA.json

```diff
@@ -403,7 +403,9 @@
 
   "profile.privateGroup": "Private group",
   "profile.connections": "Connections",
-
+  "profile.connections.tooltip1": "By clicking 'Add connection' you will be adding this user to your Private Group. To learn more, visit: ",
+  "profile.connections.tooltip2": "By clicking 'Remove connection' you will be removing this user from your Private Group. To learn more, visit: ",
+  "profile.connections.tooltip3": "To add other I-Talent users to your Private group, visit their profiles and click the 'Add connection' button on the top of their profile. To learn more about Private groups, visit: ",
```

Comment:
could you rename the keys to be more symbolic instead of 1,2,3. Maybe something like "profile.connections.tooltip.add", ...tooltip.remove", ....tooltip.info"?